test/drain_test.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	goctx "context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+var _ = Describe(
+	"NodeDrain", func() {
+		ctx := goctx.TODO()
+		Context(
+			"Clean local PVC", func() {
+				clusterName := "fake-local-volume-cluster"
+				clusterNamespacedName := getNamespacedName(
+					clusterName, namespace,
+				)
+				aeroCluster := createDummyAerospikeCluster(
+					clusterNamespacedName, 2,
+				)
+
+				It(
+					"Should delete local PVC if pod is unschedulable", func() {
+						aeroCluster.Spec.Storage.LocalStorageClasses = []string{storageClass}
+						aeroCluster.Spec.Storage.CleanLocalPVC = true
```

Comment:
We should also add a test-case where `CleanLocalPVC = false` and PVC should not be deleted (i.e. match UUID)